crates/cxx-qt-gen/src/generator/cpp/qobject.rs

```diff
@@ -37,11 +37,11 @@ impl GeneratedCppQObjectBlocks {
                 "Q_CLASSINFO(\"QML.Element\", \"{}\")",
                 qml_metadata.name
             ));
-            // TODO untested
+
             if qml_metadata.uncreatable {
                 qml_specifiers.push("Q_CLASSINFO(\"QML.Creatable\", \"false\")".to_owned());
             }
```

Comment:
It looks like there's only a test for generating the C++ for this. We could use a test actually using it with QML.

Reply:
Look in `examples/qml_features/tests/tst_singleton.qml` :-)

Reply:
Oh you are meaning for uncreatable not singleton ... :-)

Reply:
Hmm need to think of a sane way to test and show uncreatable :thinking:

Reply:
You could try using a `qml_uncreatable` QObject as a `#[qproperty]` of another QML element.

Reply:
Yup, it just then needs to be a raw pointer and the value needs to be created somewhere. Not sure if it's worth showing that or not. I've got a unit test itself checking that trying to create it fails though